src/utils/LoggingHelper.ts

```diff
@@ -0,0 +1,15 @@
+import { Log } from '../types/Log';
+import Transaction from '../types/Transaction';
+
+export default class LoggingHelper {
+
+  public static getSessionProperties(transaction: Transaction): Partial<Log> {
```

Comment:
Good idea, we can have the same approach for the Charging Station :-).
Another story!